src/util/components/CivicTextInput.js

```diff
@@ -52,7 +52,6 @@ const styles = StyleSheet.create({
     fontSize: 18,
```

Comment:
You could probably just rip this component out altogether. I'm not sure if it provides a nice abstraction of the default input with the styling we want, or if it just makes this more confusing

Reply:
We should keep a list of non-urgent things to track? "Not broken but smells funky To Do list"